scripts/scripts-1.js

```diff
@@ -0,0 +1,90 @@
+for (let i = 10; i < 26; i += 0.5) {
+
+  console.log(i);
+}
+
+const number = +prompt("Введіть число"); {
+
+  if (number % 2 !== 0 && number % 3 !== 0) {
+
+    console.log(`${number} просте число`);
+  }
+
+  else {
+    console.log(`${number} складене число`);
+  }
+}
+
+localStorage.setItem("number", number);
+
+discount = +prompt("Сума вашої покупки становить:");
+
+if (discount <= 100) {
+
+  alert(`Вітаю, ваша знижка 3%!`);
+}
+
+else if (discount <= 200) {
+
+  alert(`Вітаю, ваша знижка 5%!`);
+}
+
+else if (discount > 200) {
+
+  alert(`Вітаю, ваша знижка 7%!`);
+}
+
+localStorage.setItem("discount", discount);
+
+//таблиця множення
+
+for (i = 1; i <= 10; i++) {
```

Comment:
```suggestion
for (i = 2; i <= 9; i++) {
```
The task was `Вивести таблицю множення чисел від 2до 9 у зрозумілому форматі`